### PR TITLE
Update readme

### LINKED ISSUE #6: Update README with overview of module.
The readme should be clear enough that Alejandra, Matty and Alfonso can know where to start if they want to look around.

### PR DESCRIPTION
**Pull request by MIWdlB**
Adds an overview of the state of the repo into the README. 

Closes #6

### CODE REVIEW

**Review by cerysbm0 — APPROVED** [review]
Really clear, looks good

File: README.md [hunk 1]
```diff
@@ -1,31 +1,42 @@
...
+`utils/graph.py` Defines a basic graph, with nodes and weighted edges. Edges are assumed to be undirected but are stored as two directed edges, with weights defaulting to one.
 
+Examples are given in `utils/graphlibrary.py`.
 
+### Network creation
 
-### Poetry
-Poetry is a really nice python dependency manager. It lets you keep track of versions and easily update, unlike pip. 
-You can get it with `pip install poetry` then set up by running `poetry install` from within the directory. It will create a virtual environment and install consistent versions of things.
+Given a graph a `networke` object can be created using the `create_bipartite_network` function in `utils/bipartite_network.py`.
```
Comment (evansutcliffe) [inline README.md]: minor typo networke

**Review by evansutcliffe — APPROVED** [review]
yep good brief intro to the code

File: README.md (hunk 1 above)
Comment (MIWdlB) [reply README.md]:
```suggestion
Given a graph a `network` object can be created using the `create_bipartite_network` function in `utils/bipartite_network.py`.
```

Comment (MIWdlB) [reply README.md]: Thanks!

**Pull request merged**